server/routes/v1/user.ts

```diff
@@ -0,0 +1,42 @@
+import express, { Request, Response } from "express";
+import db from "../../models";
+import EmailService from "../../services/emailService";
+import UserController from "../../controllers/v1/user";
+
+const userRouter = express.Router();
+const eServ = new EmailService();
+const uContr = new UserController(db.User, eServ);
+
+userRouter.get(
+  "/confirmation/c=:token",
+  async (req: Request, res: Response) => {
+    const status: boolean = await uContr.confirmEmail(req.params.token);
+
+    if (status) {
+      res.status(200).send("Your email address has been confirmed. ");
+    } else {
+      res
+        .status(401)
```

Comment:
A 401 indicates that the user needs to be authenticated in order to execute the request. But they can't be authenticated without confirming their account - should be 400. The message body should also show the error message, something like: "Invalid token. Please try again."